util/config/generate-copyright

```diff
@@ -9,6 +9,8 @@ parser.add_argument('destination', help='The location to write the updated COPYR
 
 args = parser.parse_args()
 
+out_file = os.path.join(args.destination, 'COPYRIGHT')
```

Comment:
Ditto.

Reply:
> I don't think there's a major downside to having `make clean` remove them, though.

If we are OK with allowing them to be purged with `make clean` and re-written with `make`, this PR should put us in a better place W.R.T catching changes to `LICENSE` and `COPYRIGHT` and updating the `compiler/main` source files accordingly.